beam-migrate/Database/Beam/Haskell/Syntax.hs

```diff
@@ -696,6 +696,7 @@ instance IsSql92ExpressionSyntax HsExpr where
   leE q a b = hsApp (hsVar "leE")   [hsMaybe q, a, b]
 
   inE a b = hsApp (hsVar "inE") [a, hsList b]
+  inSelectE _ _ = error "inSelectE"
```

Comment:
I was going to ask why the `error` and then noticed there are already many more. :cry: